conduit/data/datasets/vision/nico_plus_plus.py

```diff
@@ -110,6 +110,8 @@ class NICOPP(CdtVisionDataset[TernarySample, Tensor, Tensor]):
     SampleType: TypeAlias = TernarySample
     Superclass: TypeAlias = NicoPPTarget
     Subclass: TypeAlias = NicoPPAttr
+    data_split_seed: ClassVar[int] = 666  # this is the seed from the paper
```

Comment:
Using the same seed won't lead to consistent splits but I guess 666 is as sensible a default as any, despite the numerological associations

Reply:
who would have thought that we're implementing a method from Satanists